integration-tests/utils/integration-utils.js

```diff
@@ -45,14 +45,17 @@ const getAnimatedDrawer = textContains => {
   return `//*[@role="dialog" and @aria-hidden="false"][contains(normalize-space(.), "${textContains}") or contains(@aria-label,"${textContains}") or @aria-labelledby=//*[contains(normalize-space(.),"${textContains}")]]`
 }
 
-const clickable = ({ text, textContains, isDescendant = false }) => {
-  const base = `(//a | //button | //*[@role="button"] | //*[@role="link"] | //*[@role="combobox"] | //*[@role="option"])`
+const clickable = ({ text, textContains, isDescendant = false, isDisabled = undefined }) => {
+  const checkDisabled = isDisabled !== undefined ? `[@aria-disabled=${isDisabled}]` : ''
+  const base = `(//a | //button | //*[@role="button"] | //*[@role="link"] | //*[@role="combobox"] | //*[@role="option"])${checkDisabled}`
+  if (isDisabled !== undefined) {
+    base += `[@aria-disabled="${isDisabled}"]`
+  }
```

Comment:
I should have included these lines in my earlier suggestion - sorry!

```suggestion
```

Reply:
No problem!